services/order.py

```diff
@@ -0,0 +1,33 @@
+from django.db import transaction
+from db.models import Order, Ticket, User
+from django.db.models.query import QuerySet
+
+
+@transaction.atomic()
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: str = None
+) -> None:
+    order = Order.objects.create(
+        user=User.objects.get(
+            username=username
+        )
+    )
+    if date:
+        order.created_at = date
+        order.save()
+    for ticket in tickets:
+        Ticket.objects.create(
+            row=ticket["row"],
+            seat=ticket["seat"],
+            movie_session_id=ticket["movie_session"],
+            order=order
+        )
+
+
+def get_orders(username: str = None) -> QuerySet:
```

Comment:
You could additionally specify that a `QuerySet` of `Order` is returned in the return type annotation.